docs/composer-authentication.md

```diff
@@ -24,6 +24,10 @@ Read-only tokens are only meant to be used with an existing composer.lock file.
 If you need to run `composer update`, then either use a token with update access or your personal access token.
 Please note that you will be charged for authentication tokens with update access as if they were user accounts.
 
+### Conductor authentication tokens
+Conductor creates short-lived authentication tokens with update access for each CI run that gets scheduled. Similar to
+organization authentication tokens, access can be restricted to any set of packages which any of the organization's teams has access to. 
```

Comment:
Actually, should we also mention _where_ access can actually be configured here?

Reply:
Good point! Missed that the other section specifies where to create the tokens. Added